skimage/color/colorlabel.py

```diff
@@ -78,9 +78,9 @@ def label2rgb(label, image=None, colors=None, alpha=0.3,
 
     Parameters
     ----------
-    label : array, shape (M, N)
+    label : array, shape (M, N) or (M, N, ...)
         Integer array of labels with the same shape as `image`.
-    image : array, shape (M, N) or (..., 3, ...), optional
+    image : array, shape (M, N) or (M, N, ...., 3), optional
```

Comment:
Seeing this change, I wonder if shouldn't add the support for the `channel_axis` argument...

Reply:
This function does have `channel_axis` already! I think the question is how to best indicate that labels is "nD spatial" while image is either "nD spatial" or "nD spatial with channels"

Reply:
I agree that the docstring as currently written implies that channels are in the last position if present, which may be misleading

Reply:
I suggest to remove the shapes from the `label` and `image` variables type and specify in the description that `image` should have the same shape as `label` with an optional additional RGB channel axis...

Reply:
a pushed a commit with the suggested update. please review